### ISSUE TITLE
buildToolsVersion not compatible with React Native 0.56

### ISSUE THREAD

**Issue by taranda** [oblador/react-native-keychain#136]
After upgrading to React Native 0.56, I get the following error when trying to compile for Android:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':react-native-keychain'.
      > The SDK Build Tools revision (23.0.1) is too low for project ':react-native-keychain'. Minimum required is 25.0.0
```

Here is the relevant portions of my `package.json`:
```
  "dependencies": {
    ...
    "react": "^16.4.1",
    "react-native": "^0.56.0",
    "react-native-keychain": "^3.0.0-rc.3"
  },
```

This may be as simple as updating `android/build.gradle` to use a later `buildToolsVersion`.    However, not being an Android developer, I am not sure what other problems upgrading the version could create.

This change is a little time sensitive, since the Google Play Store will not accept new apps after August 2018 that do not target API level 26.  React Native 0.56 targets API 26 to meet this requirement.  In other words, anyone who wants to publish a new app to Google Play in August 2018 will need to use RN 0.56 or manually target API 26.  Either way, I think this package will also need to target API 26.

**Comment by yeomann** [comment]
doesn't that each plugin on React native needs to be updated?

**Comment by taranda** [comment]
Yes...there are many react-native packages that need to be updated if they are going to work out of the box with react native 0.56.

**Comment by vonovak** [comment]
this is fixed in 3.0.0
